### ISSUE TITLE
Adapt ErrorMailHtmlFormatter to use Razor

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Benknightdark/elmah#299]
```
What new or enhanced feature are you proposing?

Adapt ErrorMailHtmlFormatter to use Razor as a templating engine to generate 
the code that generates the mail HTML.

What goal would this enhancement help you achieve?

- Easier to read and maintain a template than C# code that generates HTML
- Consistency with other views in ELMAH 2.0 that also use Razor 


```

Original issue reported on code.google.com by `azizatif` on 28 Jun 2012 at 3:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `jamesdriscoll71` on 11 Jul 2012 at 9:10
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Work under way here: http://code.google.com/r/jamesdriscoll71-issue299/
```

Original comment by `jamesdriscoll71` on 11 Jul 2012 at 9:11
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Pulled into main/default repo & thanks for the great work! :) You may 
considered the issue closed! I've continued to make improvement which you may 
want to pull back in your repo if you wish to still fine tune further.
```

Original comment by `azizatif` on 12 Jul 2012 at 12:23

**Comment by GoogleCodeExporter** [comment]
```
Closed by revision e150d64a9355
```

Original comment by `azizatif` on 2 Apr 2015 at 5:50
- Changed state: **Fixed**
